cloud.foundry.cli/src/test/java/cloud/foundry/cli/operations/ApplicationsOperationsTest.java

```diff
@@ -223,6 +216,47 @@ public void testRemoveApplicationShouldThrowNullPointerExceptionWhenApplicationN
         assertThrows(NullPointerException.class, () -> applicationsOperations.remove(null));
     }
 
+    @Test
+    public void testRenameOnCurrentNameNullThrowsNullPointerException() {
```

Comment:
I don't understand what you like to test by this method name. 
Maybe something like "testRenameWithNullValueForCurrentNameThrowsNullPointerException"

Reply:
done